extension/lib/tests/html-attribute.test.js

```diff
@@ -0,0 +1,131 @@
+const FIREFOX_1 = {
+  id: "firefox",
+  version: "1",
+};
+
+test("a supported global attribute", () => {
+  const elementName = "body";
+  const attributes = [{ name: "id", value: "test-id" }];
+  const issues = webcompat.getHTMLElementIssues(elementName, attributes, [FIREFOX_69]);
+  expect(issues.length).toBe(0);
+});
+
+test("a supported global attribute but different element", () => {
+  const elementName = "div";
+  const attributes = [{ name: "id", value: "test-id" }];
+  const issues = webcompat.getHTMLElementIssues(elementName, attributes, [FIREFOX_69]);
+  expect(issues.length).toBe(0);
+});
+
+test("a non supported html attribute", () => {
+  const elementName = "a";
+  const attributeName = "download";
+  const attributeValue = "test.png";
+  const attributes = [{ name: attributeName, value: attributeValue }];
+  const issues =
+    webcompat.getHTMLElementIssues(elementName, attributes, [FIREFOX_69, FIREFOX_1]);
+  expect(issues.length).toBe(1);
+
+  const expectedIssue = {
+    type: WebCompat.ISSUE_TYPE.HTML_ATTRIBUTE,
+    element: elementName,
+    attribute: attributeName,
+    value: attributeValue,
+    unsupportedBrowsers: [FIREFOX_1],
+  };
+  assertIssue(issues[0], expectedIssue);
+});
+
+test("an experimental html attribute", () => {
+  const elementName = "menu";
+  const attributeName = "label";
+  const attributeValue = "test";
+  const attributes = [{ name: attributeName, value: attributeValue }];
+  const issues = webcompat.getHTMLElementIssues(elementName, attributes, [FIREFOX_69]);
+  expect(issues.length).toBe(2);
+
+  const expectedElementIssue = {
+    type: WebCompat.ISSUE_TYPE.HTML_ELEMENT,
+    element: elementName,
+    experimental: true,
+    unsupportedBrowsers: [],
+  };
+  assertIssue(issues[0], expectedElementIssue);
+
+  const expectedAttributeIssue = {
+    type: WebCompat.ISSUE_TYPE.HTML_ATTRIBUTE,
+    element: elementName,
+    attribute: attributeName,
+    value: attributeValue,
+    experimental: true,
+    unsupportedBrowsers: [],
+  };
+  assertIssue(issues[1], expectedAttributeIssue);
+});
+
+test("a deprecated html attribute", () => {
+  const elementName = "table";
+  const attributeName = "align";
+  const attributeValue = "left";
+  const attributes = [{ name: attributeName, value: attributeValue }];
+  const issues = webcompat.getHTMLElementIssues(elementName, attributes, [FIREFOX_69]);
+  expect(issues.length).toBe(1);
+
+  const expectedIssue = {
+    type: WebCompat.ISSUE_TYPE.HTML_ATTRIBUTE,
+    element: elementName,
+    attribute: attributeName,
+    value: attributeValue,
+    deprecated: true,
+    unsupportedBrowsers: [],
+  };
+  assertIssue(issues[0], expectedIssue);
+});
+
+test("a data-* attribute", () => {
+  const elementName = "body";
+  const attributes = [{ name: "data-test", value: "test value" }];
+  const issues = webcompat.getHTMLElementIssues(elementName, attributes, [FIREFOX_69]);
+  expect(issues.length).toBe(0);
+});
+
+test("an invalid html element", () => {
```

Comment:
Please update this comment to explain why we ignore invalid attributes, so it's easy to understand in the future when we'll be changing this code.

`invalid html attributes are ignored because they are commonly used by frameworks`